core/src/ops/cnn/conv/unary.rs

```diff
@@ -964,7 +964,7 @@ impl TypedOp for ConvUnary {
                 && self.group == self.input_channels()
                 && input_fact.shape.as_concrete().is_some()
             {
-                let op = dispatch_floatlike!(Self::to_depth_wise(dt)(self, &input_fact))
+                let op = dispatch_numbers!(Self::to_depth_wise(dt)(self, &input_fact))
```

Comment:
Not sure about this, but it seems that there is no need for the number to be a float ?